Please refer the compiling error on Android
```
 /home/runner/work/cocos-engine/cocos-engine/native/cocos/base/threading/MessageQueue.cpp:284:5: error: use of undeclared identifier 'ADPFManager'
      ADPFManager::getInstance().AddThreadIdToHintSession(tid);
```

--- a/native/cocos/base/threading/MessageQueue.cpp
+++ b/native/cocos/base/threading/MessageQueue.cpp
@@ -278,7 +278,7 @@ MessageQueue::~MessageQueue() {
 }
 
 void MessageQueue::consumerThreadLoop() noexcept {
-#if CC_PLATFORM == CC_PLATFORM_ANDROID
+#if CC_PLATFORM == CC_PLATFORM_ANDROID && CC_SUPPORT_ADPF == 1
     // add tid to PerformanceHintManager
     int32_t tid = gettid();
     ADPFManager::getInstance().AddThreadIdToHintSession(tid);

--- a/native/cocos/platform/android/adpf_manager.cpp
+++ b/native/cocos/platform/android/adpf_manager.cpp
@@ -306,24 +306,21 @@ void ADPFManager::EndPerfHintSession(jlong target_duration_ns) {
     }
 #endif
 }
-void ADPFManager::AddThreadIdToHintSession(int32_t tid)
-{
+void ADPFManager::AddThreadIdToHintSession(int32_t tid) {
     thread_ids_.push_back(tid);
     auto data = thread_ids_.data();
 
     registerThreadIdsToHintSession();
 }
 
-void ADPFManager::RemoveThreadIdFromHintSession(int32_t tid)
-{
+void ADPFManager::RemoveThreadIdFromHintSession(int32_t tid) {
     thread_ids_.erase(std::remove(thread_ids_.begin(), thread_ids_.end(), tid), thread_ids_.end());
     auto data = thread_ids_.data();
 
     registerThreadIdsToHintSession();
 }
 
-void ADPFManager::registerThreadIdsToHintSession()
-{
+void ADPFManager::registerThreadIdsToHintSession() {
 #if __ANDROID_API__ >= 34
     auto data = thread_ids_.data();
     std::size_t size = thread_ids_.size();

